src/components/Events/EventsOverview.vue

```diff
@@ -57,10 +62,14 @@ export default {
         }
     },
     methods: {
-        getFilteredEvents(n, isMan, isLight) {
+        getFilteredEvents(n, isMan, isLight, isMix = false) {
```

Comment:
huh wordt hier niet isMix altijd op false geassigned?

Reply:
Als de arg niet mee gegeven wordt. Testen is toch wel fijn want de logica word wel pittig.